perllib/FixMyStreet/App/Form/Noise.pm

```diff
@@ -414,7 +414,7 @@ has_field radius => (
 );
 
 has_page when => (
-    fields => ['happening_now', 'happening_days', 'happening_time', 'continue'],
+    fields => ['happening_now', 'happening_pattern', 'happening_days', 'happening_time', 'happening_description', 'continue'],
```

Comment:
If we're asking a question, and the answer leads to two entirely different questions appearing, I think this would be better done as two different pages, so yes to happening_pattern would take you to a page with days and time on, and no would take you to a page with description on, and both would then take you to more_details; that way it's smaller per-page, and doesn't confuse if the JS doesn't work and all the questions get shown.